src/app/home-page/widget/starred-box/starred-box.component.scss

```diff
@@ -11,5 +11,15 @@ img[alt='User avatar'] {
 
 img[alt='Org avatar'] {
   width: 6rem;
-  height: 6rem;
+  max-height: 6rem;
+  object-fit: scale-down;
+}
+
+.size-small mat-icon {
+  color: mat.get-color-from-palette($dockstore-app-accent-1, darker);
+}
+
+// Overide mat-card padding to extend mat-divider to edges
+mat-divider {
+  margin: 0 -16px;
```

Comment:
Should this be rem instead of px? Perhaps not for a divider, just asking.

Reply:
`mat-card` by default has 16px of padding so just using px to override the existing style, would use rems but the conversion can vary which might look interesting when overriding a style

Reply:
Can you set the padding to 0 instead of using margin?